roles/debian-hardening/vars/main.yml

```diff
@@ -12,6 +12,8 @@ kernel_params:
  - slab_common.usercopy_fallback=N
  - iommu=pt
  - security=yama
+ - mce=off
```

Comment:
I was wrong in my previous comment. mce=0 does not disable MCE but sets the tolerance level to 0, in a result, all MCE events will force Linux to panic.
So mce=0 was correct.